### ISSUE TITLE
Create A New Book Page Lacks Main Landmark

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1786]
### Steps to Reproduce

1. Go to the admin interface
2. Navigate to the Toolbar 
3. Select the "Create a new book" link in the "My Catalog" sub menu
4. Inspect the ARIA landmarks for the page.

**Expected behavior:** 
This page, like most every page on the internet, should have a ```<main role="main">``` element that encompass the main region of the page, in this case everything between the banner and the complementary region.
**Actual behavior:** 
This page, like an unfortunate amount of other pages on the internet, does not have a main landmark region.

**Comment by SteelWagstaff** [comment]
We will report upstream to WordPress (can remediate with JavaScript for now).

**Comment by avaaec** [comment]
This is a UW ATS priority 0 issue

**Comment by SteelWagstaff** [comment]
Summary: the issue here is the wp-signup.php page (which we inherit from WordPress) and which is used to register new sites on a multisite network does not have a `role="main"` landmark. I found a wordpress trac issue that noted a bunch of other a11y issues for that page and added a comment there: https://core.trac.wordpress.org/ticket/40361#comment:17. While we wait to understand if/when this will be addressed in core WordPress, we should be able to remediate with javascript.

**Comment by SteelWagstaff** [comment]
There is a container div called `<div class="mu_register wp-signup-container">` that seems like a good candidate to receive the role="main" designation in this case.
![Screenshot from 2019-11-14 13-10-08](https://user-images.githubusercontent.com/13485451/68888268-36cbfa80-06e0-11ea-8e52-f68be5d2eeb0.png)

**Comment by SteelWagstaff** [comment]
@ho-man-chan I just filed a ticket w/ proposed fix upstream at WordPress: https://core.trac.wordpress.org/ticket/48657. This was my first time doing so, so who knows how well I did. Just FYI.

**Comment by connermwallace** [comment]
University of Washington Accessibility Technology Services has inspected this issue and we have verified that it has been fixed. Version: 5.3.2.
